README.md

```diff
@@ -0,0 +1,25 @@
+# terra-axon-examples
+
+## Overview
+
+This repository contains example notebooks and documentation for working with [Enterprise Terra](https://verily.com/solutions/terra/).
+
+[Enterprise Terra](https://verily.com/solutions/terra/) is a new platform developed by Verily to connect the multimodal data ecosystem and meet the needs of commercial or industry-led research organizations. Building on Terra’s Academic roots, Enterprise Terra is an integrated platform that provides the secure foundation to discover, govern, and unify multimodal data into a collaborative environment to accelerate discoveries.
+
+- Explore: discover multimodal data sets available in Enterprise Terra’s data ecosystem.
+- Unify: connect all data types, from both internal and external sources, into a single platform and under one governance framework.
+- Analyze: unlock insights faster with familiar and proprietary analysis tools.
+- Collaborate: share best practices and publish reproducible research.
+- Scale: maximize the power of any cloud with security, compliance, and support from industry standards.
+
+
+## Getting started
+
+### Setup Github integration in Enterprise Terra
+
+Go to [Enterprise Terra](https://verily.com/solutions/terra/) and:
+1. Add your SSH key to GitHub. 
+2. Create a personal workspace for yourself where you can test out many examples, such as this one. Name it something like *"Personal workspace {your ldap} {YYYYMMDD}"*. (You don't need to have multiple workspaces when you are just trying stuff out on public data, but if you want multiple workspaces, of course feel free to create them!)
```

Comment:
Maybe leave out 'for yourself' as that's what 'personal' implies.  

'ldap' is pretty Alphabet-specific - is this intended for an Alphabet-only audience?  Maybe email address instead?

I'd separate out the parenthetical phrase into a note.  Also, maybe something like, 'You don't need multiple workspaces to work on public data, but you can create as many as you'd like.'  Something to that effect.